proxyvisor/proxyvisor.go

```diff
@@ -0,0 +1,19 @@
+package proxyvisor
+
+func Provision() (string, error) {
+	return "PIo0c6fYkQN0liVOPAVs7WA4WtP8NYChlDjRKPO75xpmT1Kj0wk1nPJFIwDStv", nil
+	// resp, err := http.Post("http://localhost:3000/v1/device", "", nil)
```

Comment:
A nitty thing, but really hate seeing blocks of commented-out code checked in. A reviewer/new maintainer doesn't know the context - is this just old code? Is it an example? Can I remove it? You can end up with lots of rotting code this way. Consider a branch as an alternative.